recipes/icclim/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "icclim" %}
+{% set version = "5.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/icclim-{{ version }}.tar.gz
+  sha256: 66b3c0202b3e2a9ac6510120bf52452680185c832e48fb0dea9e61447c91b9c9
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - xarray >=0.19
+    - xclim >=0.33
+  run:
+    - python >=3.8
+    - xclim >=0.33
+    - numpy >=1.21.1
+    - xarray >=0.19
+    - dask[array] >=2021.10.0
+    - netCDF4 >=1.5.7
+    - cftime >=1.5.0
+    - pyyaml >=6.0
+    - cftime >=1.4.1
+    - setuptools-git >=1.2
+
+test:
+  imports:
+    - icclim
+    - icclim.models
+  commands:
+    - pip check
+  source_files:
+    - tests/unit_tests
+  requires:
+    - pip
+    - pytest
+
+about:
+  summary: 'icclim (Index Calculation CLIMate) is a python library for climate indices calculation.'
+  home: https://github.com/cerfacs-globc/icclim
+  doc_url: https://icclim.readthedocs.io/
+  dev_url: https://github.com/cerfacs-globc/icclim
+  license: Apache-2.0
+  license_file: LICENCE
```

Comment:
I think you need to change the packaging of the tarball slightly. The file `LICENSE` (sic, i.e. with an S to conform to American English) should contain the actual license, i.e. be a copy of https://www.apache.org/licenses/LICENSE-2.0.txt. What you have in `LICENCE` right now might be appropriately placed in `NOTICE`. What you have in `NOTICE` can go to `README` or anything else. More info about those files from the perspective of the Apache Foundation can be found at https://infra.apache.org/licensing-howto.html

There may be some room for interpretation, but one thing is clear: The conda-forge distribution *must* contain the actual license, not just a pointer to it.

Reply:
This is very informative, thank you !